README.md

```diff
@@ -10,6 +10,42 @@ Clusterpedia can synchronize resources with multiple clusters and provide more p
 
 > The capability of Clusterpedia is not only to search for and view but also simply control resources in the future, just like Wikipedia that supports for editing entries.
 
+## Why Clusterpedia
+Clusterpedia can be deployed and used as a standalone platform, or it can be integrated with [Cluster API](https://github.com/kubernetes-sigs/cluster-api), [Karmada](https://github.com/karmada-io/karmada), [Clusternet](https://github.com/clusternet/clusternet) and user-developed multi-cloud platforms
+> Q2 also automaatically synchronizes resources within clusters managed by `Cluster API`, `Karmada`, `Clusternet` and self-developed multi-cloud platforms.
+
+### Rich retrieval features and compatibility with **Kubernetes OpenAPI**
+* Support for retrieving resources using `kubectl`, `client-go` or `controller-runtime/client`, [client-go example](https://github.com/clusterpedia-io/client-go/blob/main/examples/list-clusterpedia-resources/main.go)
+* The metadata of the resource an be retrived via API or [client-go/metadata](https://pkg.go.dev/k8s.io/client-go/metadata)
+* Rich retrieval conditions: [Filter by cluster/namespace/name/creation](https://clusterpedia.io/docs/usage/search/multi-cluster/#basic-features), [Search by parent or ancestor owner](https://clusterpedia.io/docs/usage/search/multi-cluster/#search-by-parent-or-ancestor-owner),[Multi-Cluster Label Selector](https://clusterpedia.io/docs/usage/search/#label-selector), [Enhanced Field Selector](https://clusterpedia.io/docs/usage/search/#field-selector), [Custom Search Conditions](https://clusterpedia.io/docs/usage/search/#advanced-searchcustom-conditional-search), etc.
+### Support for importing 1.10+ versions of Kubernetes
+### Automic conversion of different versions of Kube resources and support for multiple version of resources
+* Even if you import different version of Kube, we can still use the same resource version to retrieve resources
```

Comment:
Even if you use different Kubernetes versions, you can still retrieve resources with a same version through Clusterpedia.